### PR TITLE
Simplify the split edit code

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1904** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1904" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1904?utm_source=stack-comment).

File: src/pages/create/forms/FundingForm/DistributionSplitsSection/types.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as Moment from 'moment'
+
+export type SplitType = 'project' | 'address'
+
+export type AddOrEditSplitFormFields = {
+  projectId: string
+  beneficiary: string
+  percent: number
+  amount: string | undefined
+  lockedUntil: Moment.Moment | undefined | null
```
Comment (tomquirk) [inline src/pages/create/forms/FundingForm/DistributionSplitsSection/types.ts:10]: **question** Is this really a date? I thought it was a `number`

File: src/pages/create/forms/FundingForm/DistributionSplitsSection/DistributionSplitModal.tsx [hunk 2]
```diff
@@ -414,108 +362,17 @@ export default function DistributionSplitModal({
...
-                      raise.
-                    </Trans>
-                  }
-                  placement={'topLeft'}
-                  iconStyle={{ marginLeft: 5 }}
-                />
-              </div>
-            </div>
-          </Form.Item>
+          <AmountFormItem
```
Comment (tomquirk) [inline src/pages/create/forms/FundingForm/DistributionSplitsSection/DistributionSplitModal.tsx:365]: **praise** fantastiiicccc

**Review by tomquirk — APPROVED** [review]
@wraeth-eth approving in advance. left a comment or two <3

File: src/pages/create/forms/FundingForm/DistributionSplitsSection/types.ts (hunk 1 above)
Comment (wraeth-eth) [reply src/pages/create/forms/FundingForm/DistributionSplitsSection/types.ts:10]: Yeah definitely not my first pick, but it is workjng for now so afraid to change it much

**Pull request merged**